youtube_dl/extractor/mixcloud.py

```diff
@@ -3,16 +3,20 @@
 import re
 
 from .common import InfoExtractor
-from ..compat import compat_urllib_parse_unquote
+from ..compat import (
+    compat_urllib_parse_unquote,
+    compat_urllib_request
+)
 from ..utils import (
     ExtractorError,
     HEADRequest,
     str_to_int,
+    clean_html
 )
 
 
 class MixcloudIE(InfoExtractor):
-    _VALID_URL = r'^(?:https?://)?(?:www\.)?mixcloud\.com/([^/]+)/([^/]+)'
+    _VALID_URL = r'^(?:https?://)?(?:www\.)?mixcloud\.com/([^/]+)/(?!stream|uploads|favorites|listens|playlists)([^/]+)'
```

Comment:
I did not implement support for `stream`. I still excluded it, because `MixcloudIE` can't handle it anyways. I doubt that #7633 is using this code, as this PR is not merged.